zou/app/services/assets_service.py

```diff
@@ -258,6 +259,15 @@ def get_assets_and_tasks(criterions={}, page=1, with_episode_ids=False):
                 str(link.entity_in_id)
             )
 
+    query_result = tasks_query.all()
+
+    if "vendor_departments" in criterions:
+        not_allowed_descriptors_field_names = entities_service.get_not_allowed_descriptors_fields_for_entity_type_and_departments(
```

Comment:
* The function name is too long
* The line limit width is 80 chars

Reply:
It's fixed